.github/workflows/performance-tests.yml

```diff
@@ -90,23 +87,29 @@ jobs:
           scope: 'input-output-hk'
 
       - name: Install dependencies
-        uses: borales/actions-yarn@v4
+        uses: borales/actions-yarn@v4.2.0
         with:
           cmd: install
           dir: ${{ env.BENCHMARKING_DIR }}
 
       - name: Compile tests to JS
-        uses: borales/actions-yarn@v4
+        uses: borales/actions-yarn@v4.2.0
         with:
           cmd: webpack
           dir: ${{ env.BENCHMARKING_DIR }}
 
-      - name: Connection Flow Smoke Test
+      - name: All Smoke Tests
         env:
           ISSUER_AGENT_API_KEY: default
```

Comment:
@davidpoltorak-io, you need to make the length of the key at least 16 bytes. I will not work with the current Agent with `default` value.

Reply:
Thanks @yshyn-iohk - I've been struggling to get this branch ready for merging - between performance thresholds being passed and some failed runs - this really helps - I'll make the change and retest